en/components/date_range_picker.md

```diff
@@ -58,71 +58,52 @@ To create a two-way data-binding, use `ngModel`:
 
 ```html
 <igx-date-range-picker [(ngModel)]="range">
     <igx-date-range-start>
         <input igxInput igxDateTimeEditor type="text">
-        <igx-picker-toggle igxPrefix>
-            <igx-icon>calendar_today</igx-icon>
-        </igx-picker-toggle>
     </igx-date-range-start>
     <igx-date-range-end>
         <input igxInput igxDateTimeEditor type="text">
     </igx-date-range-end>
 </igx-date-range-picker>
 ```
+- Both the [`IgxDateRangeStartComponent`]({environment:angularApiUrl}/classes/igxdaterangestartcomponent.html) and [`IgxDateRangeEndComponent`]({environment:angularApiUrl}/classes/igxdaterangeendcomponent.html) extend the existing [`IgxInputGroupComponent`](input_group.md). For such a configuration to work, defining an [`IgxInput`]({environment:angularApiUrl}/classes/igxinputdirective.html) is required. In addition, all other components and directives available to the [`IgxInputGroupComponent`](input_group.md) can also be used.
+- Another thing worth mentioning is that in order to enable date editing for both inputs, you need to decorate them with [`igxDateTimeEditor`](date_time_editor.md) directive.
+
+<div class="sample-container loading" style="height:600px">
+    <iframe id="daterangepicker-start-end-iframe" src="{environment:demosBaseUrl}/scheduling/daterangepicker-start-end" width="100%" height="100%" seamless="" frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="daterangepicker-start-end-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
 
 #### User Experience
 
 In a default configuration, with a single read-only input, the calendar can be opened by clicking anywhere in the input, including the calendar icon. When there are two separate inputs for start and end date, the calendar can only be opened from the calendar icon, since both inputs are editable by default.
 
-When the calendar is visible, a range can be selected by choosing the start and end dates. Picking a date will set both start and end date, until a second date is chosen. If there is a selected range including more than one day, clicking any date from the calendar will result in selecting that date for both start and end date. Clicking on a day that represents both a start and an end of the current range, will result in deselecting that single date range.
+When the calendar is visible, a range can be selected by choosing the start and end dates. Picking a date will set the start and end date, until a second date is chosen. If there is a selected range, clicking any other date from the calendar will start a new range selection. If a single date is selected, it will count as the start and end of the current range selection, clicking on any other date will start a new range selection while clicking on it will remove the range selection.
```

Comment:
"If a single date is selected, it will count as the start and end of the current range selection" (Thinking,  if a single date is a "range selection", then the previous statement is incorrect, as a single date is a range by itself) --> If there is a selected range, clicking any other date from the calendar will start a new range selection. (it will update/end the range selection rather than starting a new one)